rosbag2_cpp/include/rosbag2_cpp/readers/sequential_reader.hpp

```diff
@@ -101,7 +101,6 @@ class ROSBAG2_CPP_PUBLIC SequentialReader
   */
   virtual void load_next_file();
 
-private:
```

Comment:
needed? could it just be protected?

Reply:
This changes the following from private to protected, the next one up is `protected:`